ksql-examples/src/main/resources/datagen.properties

```diff
@@ -0,0 +1,2 @@
+
+interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
```

Comment:
Am I right in thinking that `MonitoringProducerInterceptor` will only be available on the class path for enterprise users?  Won't this cause problems / exceptions for non-enterprise users? Or is the idea that only enterprise users would use this properties file?

Reply:
Yes, this will only be used when we have CE.

Reply:
Should this be documented? Even if it is just a comment in the properties file?

Reply:
Good point. I'm adding a comment in the properties file.